RevenueCatUI/CustomerCenter/Data/CustomerCenterConfigData+Mock.swift

```diff
@@ -131,6 +131,7 @@ extension CustomerCenterConfigData {
                 displayVirtualCurrencies: displayVirtualCurrencies,
                 shouldWarnCustomersAboutMultipleSubscriptions: shouldWarnCustomersAboutMultipleSubscriptions
             ),
+            changePlan: [],
```

Comment:
If this is a unique change plan, why is it a list (`[]`)

Reply:
The config has one `ChangePlan` per subscription group. Each purchase has a single `ChangePlan`

Reply:
This is what I meant https://github.com/RevenueCat/purchases-ios/pull/5379#discussion_r2210399295